packages/slice-machine/pages/[lib]/[sliceName]/[variation]/screenshot.tsx

```diff
@@ -29,7 +29,7 @@ const Screenshot: ComponentWithSliceProps = ({ slice, variation }) => {
     variationID: variation.id,
   });
 
-  if (!simulatorUrl) {
+  if (simulatorUrl === undefined) {
```

Comment:
can simulator url be an other falsely value?

Reply:
no but Typescript complains if you use a non-boolean value in conditions